src/spicy/spicyz/glue-compiler.cc

```diff
@@ -1318,7 +1318,9 @@ struct VisitorZeekType : hilti::visitor::PreOrder<hilti::Result<hilti::Expressio
                 SPICY_DEBUG(hilti::util::fmt("Creating Zeek record type %s::%s with fields:", ns, local));
 
                 for ( const auto& f : fields )
-                    SPICY_DEBUG(hilti::util::fmt("  %s", f.as<hilti::expression::Ctor>()
+                    SPICY_DEBUG(hilti::util::fmt("  %s", f.as<hilti::expression::UnresolvedOperator>()
+                                                             .operands()[1]
+                                                             .as<hilti::expression::Ctor>()
```

Comment:
I'm wondering if we should just print the original expression `f` here, instead of trying to decompose it, which seems brittle in case of further changes in the future.

Reply:
Yeah, not super pretty, but seems fine to me.